consensus/src/model/stores/headers.rs

```diff
@@ -25,6 +27,14 @@ pub struct HeaderWithBlockLevel {
     pub block_level: BlockLevel,
 }
 
+impl MemSizeEstimator for HeaderWithBlockLevel {
+    fn estimate_mem_size(&self) -> MemSize {
```

Comment:
Please add comments why this is the correct estimation for the memsize of this object